src/core/components/hv-screen/types.ts

```diff
@@ -60,19 +63,26 @@ export type DataProps = {
  * Props used by legacy external navigation components
  */
 export type ActionProps = {
-  back?: () => void;
-  closeModal?: () => void;
+  back?: (params: object) => void;
+  closeModal?: (params: object) => void;
```

Comment:
don't we know the exact types of params we pass to back/closeModal?

Reply:
We do but I wanted to be consistent with the other implementations. If you'd prefer I strong type them, I'll switch them back. This seemed the best way to continue to support legacy functionality.

Reply:
Updated to support strong or weak typed